docs/src/main/mdoc/consumers.md

```diff
@@ -0,0 +1,314 @@
+---
+id: consumers
+title: Consumers
+---
+
+Consumers support subscribing to topics, record streaming and deserialization, as well as miscellaneous utility functionality, such as seeking to offsets, or checking what the end offsets are for a topic. Consumers make use of the [Java Kafka consumer][java-kafka-consumer], which becomes especially important for [settings](#settings). For consumer implementation details, refer to the [technical details](technical-details.md) section.
```

Comment:
why is it especially important for settings?

Reply:
Because settings (`ConsumerSettings`) include settings for the Java Kafka consumer.